### ISSUE TITLE
Support relative font size literals (rem)

### ISSUE THREAD

**Issue by fvarano** [kizeevov/slint-idea-plugin#18]
The sling language supports passing font-sizes as a factor relative to the root font size of the window ([reference](https://slint.dev/releases/1.3.0/docs/slint/src/language/syntax/types)).

Example:

```slint
import { Button, VerticalBox } from "std-widgets.slint";

export component MainWindow inherits Window {
    in-out property<int> counter: 42;
    default-font-size: 14px;
  
    VerticalBox {
        Text {
            text: "Counter: \{root.counter}";
            font-size: 1.2rem;
            horizontal-alignment: center;
        }
        Button {
            text: "Increase value";
            clicked => {
                counter += 1;
            }
        }
    }
}
```

Currently this plugin does not support it.

It may just be another LengthLiteral defined [here](https://github.com/kizeevov/slint-idea-plugin/blob/4fded812ae08a70815117d6e6de7392691fc2fbd/src/main/grammars/SlintParser.bnf#L533C9-L533C9).

Thanks for this plugin!

**Comment by picrap** [comment]
Fun fact: it seems to be implemented since 3 weeks, but just waits to be released 😉

**Comment by kizeevov** [comment]
Yes, I'm sorry for the delay. Moderation in jb takes some time. And I wanted to combine the fix with the new features. 
The fix is available in version 0.3.1

**Comment by picrap** [comment]
I saw it yesterday, along with the new preview feature 🤩. Thank you.
